core/src/test/scala/org/bykn/bosatsu/NormalizationTest.scala

```diff
@@ -0,0 +1,79 @@
+package org.bykn.bosatsu
+
+import cats.data.{ Validated, ValidatedNel}
+import fastparse.all._
+import org.scalatest.FunSuite
+
+class NormalizationTest extends FunSuite {
+
+  def resolveThenInfer(ps: Iterable[Package.Parsed]): ValidatedNel[PackageError, PackageMap.Inferred] =
```

Comment:
I have a `object TestUtils` that I have been trying to centralize utilities for testing to make it easier to get going in tests. I think we should see what is there and if these aren't, move them there. I let some of the test code get out of control a bit...

Reply:
cool. I'm going to try to fill out the tests this weekend